### DISCUSSION TITLE
Docker exec

### DISCUSSION THREAD

**Discussion by wendyliga** [buildbuddy-io/buildbuddy#2369] [Q&A] [answered]
does buildbuddy's docker image not have any bash support?

running this to exec command on the container
```
docker exec -it buildbuddy "/bin/sh"
```

got this 
```
OCI runtime exec failed: exec failed: unable to start container process: exec: "/bin/sh": stat /bin/sh: no such file or directory: unknown
```

**Comment by siggisim** [comment, marked as answer]
Hey Wendy, 

We run a stripped down "distroless" base image which only contains our application and our runtime dependencies. This greatly reduces both the size of the image and improves our security posture. You can read more about distroless images here: https://github.com/GoogleContainerTools/distroless

If you'd like to build your own image with whatever tools you'd like on it - you can swap out this base image here: https://github.com/buildbuddy-io/buildbuddy/blob/ca04893073cb4cc2be233102d1cfd30e46b534db/WORKSPACE#L232

And build the image here: https://github.com/buildbuddy-io/buildbuddy/blob/94d9d955344da92a6a6363523bbf3367e71eb4ca/server/cmd/buildbuddy/BUILD#L40

\- Siggi

**Reply by wendyliga** [reply]
that's makes sense, thanks for the info
